ports/raspberrypi/boards/waveshare_rp2040_plus/mpconfigboard.mk

```
@@ -0,0 +1,11 @@
+USB_VID = 0x2E8A
+USB_PID = 0x101D
```

```suggestion
USB_PID = 0x0005
```

Apologies, forgot to set the original PID back after testing